### ISSUE TITLE
On Windows, do not ship unnecessary/extra binaries

### ISSUE THREAD

**Issue by steazzalini** [kiitoscpp/llvm#1]
Linux distributions only ships `llvm-config` and `llvm-tblgen`: the same should apply to Windows distributions.

**Comment by steazzalini** [comment]
This is a working script that could be used:

```batch
@echo off

for %%i in (bin\*) do (
  if "%%~xi" == ".exe" (
    if not "%%~ni" == "llvm-config" (
      if not "%%~ni" == "llvm-tblgen" (
        del "%%i"
      )
    )
  )
)
```

**Comment by steazzalini** [comment]
Fixed by feb253ac0f7d409f4cd090ff8239736850a63892.
